Weather/Modules/WeatherLocation/Core/View/WeatherLocationViewController.swift

```diff
@@ -1,18 +1,126 @@
 import UIKit
+import ASToast
 
-class WeatherLocationViewController: UIViewController, WeatherLocationView {
 
+class WeatherLocationViewController: UIViewController,
```

Comment:
We should have extensions for protocol conformance in all our VCs, or at least where there is a lot of code .. 

https://github.com/raywenderlich/swift-style-guide#code-organization